cdist/test/exec/local.py

```diff
@@ -35,17 +36,21 @@
 fixtures = op.join(my_dir, 'fixtures')
 conf_dir = op.join(fixtures, "conf")
 
+bin_true = "/usr/bin/true"
+bin_false = "/usr/bin/false"
```

Comment:
I'm not a big fan of hardcoding paths for true/false  (like we did before) - maybe use PATH resolution for it?